src/rebar3_formatter.erl

```diff
@@ -18,27 +18,33 @@
 -spec format(file:filename_all(), module(), opts()) -> result().
 format(File, Formatter, Opts) ->
     AST = get_ast(File),
-    QuickAST = get_quick_ast(File),
     Comments = get_comments(File),
-    FileOpts = apply_per_file_opts(File, Opts),
-    {ok, Original} = file:read_file(File),
-    Formatted = format(File, AST, Formatter, Comments, FileOpts),
-    Result = case Formatted of
-               Original ->
-                   unchanged;
-               _ ->
-                   changed
-             end,
-    case maybe_save_file(maps:get(output_dir, FileOpts), File, Formatted) of
-      none ->
-          Result;
-      NewFile ->
-          case get_quick_ast(NewFile) of
-            QuickAST ->
-                Result;
-            _ ->
-                erlang:error({modified_ast, File, NewFile})
-          end
+    case apply_per_file_opts(File, Opts) of
+        ignore ->
+            unchanged;
```

Comment:
We might consider (maybe in a separate ticket) to change this to `ignored`.

Reply:
Actually… shouldn't we write the file, but unchanged, in the destination folder? I mean… if it's `current`, then it doesn't really matter… but if they want to dump the formatted files somewhere else (like we do in our tests)… I guess ignored files should still be put there… just not modified.